arch/risc-v/src/bl602/Kconfig

```diff
@@ -34,4 +34,35 @@ config BL602_TIMER0
 
 config BL602_TIMER1
 	bool "TIMER1"
+
+config BL602_PWM0
+	bool "PWM0"
+	default y
+    depends on PWM 
+    select PWM_MULTICHAN
+
 endmenu
+
+config BL602_SPIFLASH
```

Comment:
@liangzhanggb Please split the SPI related change to another PR.

Reply:
Thank you for your reminder, I will delete it immediately